lib/teleterm/clusters/storage.go

```diff
@@ -264,6 +264,16 @@ func (s *Storage) makeDefaultClientConfig() *client.Config {
 	cfg.KeysDir = s.Dir
 	cfg.InsecureSkipVerify = s.InsecureSkipVerify
 	cfg.WebauthnLogin = s.WebauthnLogin
+	// Set AllowStdinHijack to true to enable daemon.mfaPrompt to ask for both TOTP and Webauthn at
+	// the same time if available.
+	//
+	// tsh sets AllowStdinHijack to true only during tsh login to avoid input swallowing bugs where
+	// calling a command would prompt for MFA and then expect some further data through stdin. tsh
+	// login does not ask for any further input after the MFA prompt.
+	//
+	// Since tsh daemon ran by Connect never expects data over stdin, it can always set this flag to
+	// true.
+	cfg.AllowStdinHijack = true
```

Comment:
@Joerger @codingllama I could use your help here to double check this.

This PR shows a modal in Connect when tsh daemon uses `daemon.mfaPrompt` implemented by Brian. The problem I had to solve with this one-line change is that if both Webauthn and TOTP are supported MFA methods, Connect would always prompt only for Webauthn. This is because Connect doesn't set `AllowStdinHijack` and `lib/client.mfa.PromptConfig.GetRunOptions` depends on that setting to enable Webauthn + TOTP at the same time in tsh.

tsh has `--mfa-mode` to work around if the user wants to force TOTP in a particular scenario. Connect should generally show both options and let the user pick one.

At first, the comment above `AllowStdinHijack` made me scared of using this field. But now that I looked around the codebase and understand a little bit better how it is used, I think it should be okay to set it for the Teleport client used by tsh daemon.

Reply:
Yeah, if you don't rely on reading from stdin this should be alright. We get into trouble in situations where we have to "cancel" term.ReadPassword, plus mix plain and password reads.